docs/build/2-Dependencies-install.md

```diff
@@ -53,20 +53,34 @@ The install procedure can be done
     sudo yum install cmake3 devtoolset-9
     sudo yum install libuuid-devel unzip wxGTK3-devel boost-test boost-devel
     ```
-=== "Ubuntu (apt-get)"
+=== "Ubuntu 20.04 or 22.04 / Debian 11 (apt-get)"
 
     ```
     sudo apt-get install uuid-dev libwxgtk3.0-gtk3-dev
     sudo apt-get install libboost-test-dev
     ```
 
+=== "Ubuntu 23.04 / Debian 12 (apt-get)"
+
+    ```
+    sudo apt-get install uuid-dev libwxgtk3.2-dev libboost-test-dev
+    ```
+
 ## Automatic libraries compilation from git
 [Antares dependencies compilation repository](https://github.com/AntaresSimulatorTeam/antares-deps) is used as a git submodule for automatic libraries compilation from git.
 
 Apart from OR-Tools, all dependencies can be built at configure time using the option `-DBUILD_ALL=ON` (`OFF` by default). For a list of available option see [Antares dependencies compilation repository](https://github.com/AntaresSimulatorTeam/antares-deps).
 
 You can set `-DBUILD_ORTOOLS=ON` to download & build OR-Tools. It is also possible to use a precompiled archive, see below.
 
+For compiling the package yourself from git, additional build dependencies are needed (see [here](1-Development-requirements.md)).
+
+For building on Debian 12 or Ubuntu 23.04 the installation requires the following build and test dependencies:
+
+```
+sudo apt-get install git unzip cmake build-essentials uuid-dev libboost-test-dev libwxgtk3.2-dev python3-numpy python3-pytest
+```
+
```

Comment:
This part seems redundant, is it possible to remove it ?

Reply:
I find it of value to provide a "install a dependencies"-command instead of going through 1-Development-Requirements one by one..?
As the other one is also missing some things like `unzip` and uses pip instead of built-in python3-packages

But sure.
You can also just give this PR the final polishment to get merged yourself - which is probably easier :)

Reply:
I agree that knowing all of what's required to build is indeed very useful. I suggest moving it to the relevant parts of file **1-Development-requirements.md**, that is sub-section "Ubuntu/Debian", under section C++ version.

Many thanks